datamol/__init__.py

```diff
@@ -17,6 +17,8 @@
 from .utils import parallelized_with_batches
 from .utils import JobRunner
 from .utils import fs
+from .utils import open_dataframe
+from .utils import save_dataframe
```

Comment:
I would rename to `open/save_df` instead to be consistent with the other df related datamol' functions.